code/modules/reagents/chemistry/reagents/alcohol.dm

```diff
@@ -422,7 +432,7 @@
 	drink_icon = "gargleblasterglass"
 	drink_name = "Pan-Galactic Gargle Blaster"
 	drink_desc = "Does... does this mean that Arthur and Ford are on the station? Oh joy."
-	taste_message = "the number fourty two"
```

Comment:
RIP reference

Reply:
It's still the same reference!

Reply:
Long live the reference

Reply:
I just added the actual quote to the taste message, from my own copy of the Ultimate Hitchhiker's Guide to the Galaxy.